docs/user/library/render/wkt.rst

```diff
@@ -155,3 +155,86 @@ Where *15* indicates the wind intensity and *[kts]* indicates its unit of measur
 And it produces:
 
   .. image:: /images/barbs.png
+  
+WKT Marks as Strokes
+--------------------
+
+The WKT marks are normally converted into images and then repeated along the line geometry,
+in case of continue shapes (e.g a Wave like Shape), the bends across different line segments
+will break the continuity. If the goal is to have the wkt shape repeated and connected
+irrespective of angles between line segments then it is possible to convert the WKT shape
+into a stroke instead of image. 
+
+To enable this make the following vendor option child of <LineSymbolizer> ::
+  
+  <VendorOption name="markAlongLine">true</VendorOption>					
+  ...
+
+(Optional) `markAlongLineScaleLimit` is used define the scale limit.
+
+ Valid values are between 0 and 1
+ 
+ 0 = Always Scale
+ 
+ 1 = Never Scale
+ 
+ Default value = 0.9
+ 
+ This value is used by Stroke to adjust the Shape width to its respective line segment.
+ 
+ 0.9 means that Shape cannot shrink beyond this scale limit to fit on the underlying line segment ::
+ 
+  
+  <VendorOption name="markAlongLineScaleLimit">0.9</VendorOption>
+  ...
+  
+(Optional) `markAlongLineSimplifyTolerance` is used to decide if linear geometry needs to be simplified.
+
+ Valid values are between 0 and 1
+ 
+ 0 = Never simplify
+ 
+ 1 = Always simplify
+ 
+ Default value = 0.4
+ 
+ This value represents the percentage as ratio of line segments smaller than height of WKT Shape
+ 
+ 0.4 means that if more than 40% segments of linear geometry are smaller than  height of WKT Shape,
+ then to cleanly fit WKT Shape over the linear geometry, the linear geometry will need to be simplified ::
+ 
+  
+  <VendorOption name="markAlongLineSimplifyTolerance">0.4</VendorOption>
```

Comment:
markAlongLineSimplifyTolerance is a really long name... can we use just markAlongLineSimplify?

Reply:
will change